hack/crd-schema-checker.sh

```diff
@@ -16,6 +16,7 @@ for crd in config/crd/bases/*.yaml; do
     mkdir -p "$(dirname "$TMP_DIR/$crd")"
     git show "$BASE_REF:$crd" > "$TMP_DIR/$crd"
     $CHECKER check-manifests \
+        --disabled-validators=NoBools,NoNewRequiredFields \
```

Comment:
OK - so this was just for testing.  We want to remove this and then explicitly waive the test failures.